packages/shared/src/platform.ts

```diff
@@ -27,6 +27,7 @@ export function isMac(): boolean {
     return testPlatform(/^Mac/);
 }
 
+/* c8 ignore next 3 */
```

Comment:
`isIOS` uses this method, so we should be able to leave this off and get a little better code coverage for it.